src/coreclr/vm/jithelpers.cpp

```diff
@@ -5671,6 +5671,34 @@ HCIMPLEND_RAW
 EXTERN_C void JIT_ValidateIndirectCall();
 EXTERN_C void JIT_DispatchIndirectCall();
 
+HCIMPL2_VV(void*, JIT_DispatchVirtualStatic, MethodTable *constrainedType, MethodDesc *interfaceMethod)
+{
+    FCALL_CONTRACT;
+
+    BOOL uniqueResolution;
+
+    FC_CAN_TRIGGER_GC();
```

Comment:
There needs to be `HELPER_METHOD_FRAME` around the call to `ResolveVirtualStaticMethod`

Reply:
Thanks for pointing that out, fixed.